app/modules/transactions/helpers.js

```diff
@@ -1,48 +1,33 @@
 import FilteredCollection from 'backbone-filtered-collection'
-import log_ from '#lib/loggers'
 import preq from '#lib/preq'
-import Message from './models/message.js'
-import poster_ from '#lib/poster'
 
 export default function () {
   app.reqres.setHandlers({
     'transactions:add': API.addTransaction,
     'get:transaction:byId': API.getTransaction,
-    'transaction:post:message': API.postMessage
   })
 
   return app.request('wait:for', 'user').then(initLateHelpers)
 }
 
 const API = {
   addTransaction (transaction) { return app.transactions.add(transaction) },
```

Comment:
when i want to send a request from an item modal, i get this error
![2024-01-30_21-43](https://github.com/inventaire/inventaire-client/assets/5363918/3e8a4e28-b282-4400-9039-3adf36bbdefb)
![2024-01-30_21-46](https://github.com/inventaire/inventaire-client/assets/5363918/968075f7-5f05-4f0f-8d4d-7adc696ce5cf)

Reply:
This should have been fixed by 74c48d223